gui/wxpython/datacatalog/tree.py

```diff
@@ -801,6 +804,18 @@ def InsertLayer(self, name, mapset_node, element_name):
             self._model.SortChildren(found_element)
             self.RefreshNode(mapset_node, recursive=True)
 
+    def InsertMapset(self, name, location_node, mapset_name):
+        """Insert mapset into model and refresh tree"""
+        found_element = self._model.SearchNodes(
+            parent=location_node, type='mapset', name=mapset_name)
+        found_element = found_element[0] if found_element else None
+        found = self._model.SearchNodes(parent=found_element, name=name)
+        if len(found) == 0:
+            self._model.AppendNode(parent=found_element, label=name,
```

Comment:
parent is parent node (here location node), label is mapset name and type is type of node ('mapset' here)

Reply:
I have problems with parameter "name" and "mapset_name" here:
            self.InsertMapset(name=mapset,
                              location_node=self.selected_location[0],
                              mapset_name=mapset)
I know that it is not possible to have two same parameters but do not know what to insert for 'name' . 
Now I am in this situation:
I can create and refresh mapset but it the tree has another branch:
![Screenshot from 2020-06-28 01-46-27](https://user-images.githubusercontent.com/49241681/85940380-967abd00-b8e1-11ea-8692-df5a19f68819.png)
I spent several hours on that and do not know how to solve it. I is probably simple but I cannot figure that out.

Reply:
I am not sure what's the problem here, it is pretty simple:

```
    def InsertMapset(self, name, location_node):
        """Insert mapset into model and refresh tree"""
        self._model.AppendNode(parent=location_node, label=name,
                               data=dict(type='mapset', name=name))
        self._model.SortChildren(location_node)
        self.RefreshNode(location_node, recursive=True)
```